client/src/components/home/slideshow.js

```diff
@@ -0,0 +1,84 @@
+import "./slideshow.styl";
+import React from "react";
+
+import { times } from "lodash";
+import classnames from "classnames";
+
+class Slideshow extends React.Component {
+  static propTypes = {
+    sources: React.PropTypes.array.isRequired
+  };
+
+  state = {
+    currentSlide: 0
+  };
+
+  prev() {
+    if (this.state.currentSlide === 0) {
+      this.setState({
+        currentSlide: this.props.sources.length - 1
+      });
+    } else {
+      this.setState({
+        currentSlide: this.state.currentSlide - 1
+      });
+    }
+  }
+
+  next() {
+    if (this.state.currentSlide === this.props.sources.length -1) {
+      this.setState({
+        currentSlide: 0
+      });
+    } else {
+      this.setState({
+        currentSlide: this.state.currentSlide + 1
+      });
+    }
+  }
+
+  goToSlide(i) {
+    this.setState({
+      currentSlide: i
+    });
+  }
+
+  render() {
+    return (
+      <div className="Slideshow">
+        <div className="Slideshow-prev" onClick={this.prev.bind(this)}>
+          <div className="arrow-prev">Prev</div>
+        </div>
+        <div className="Slideshow-images">
+          {times(this.props.sources.length, i => {
```

Comment:
`import { map, times } from "lodash";`

`{map(this.props.sources, (src, i) => {`